app/assets/stylesheets/articles.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the articles controller here.
```

Comment:
remove all `.scss` files, except for `static_pages.scss`, since rails only takes care of the API side and doesn't need css (expect for angular)